src/panels/config/zone/dialog-home-zone-detail.ts

```diff
@@ -0,0 +1,150 @@
+import "@material/mwc-button";
+import { css, CSSResultGroup, html, LitElement, nothing } from "lit";
+import { property, state } from "lit/decorators";
+import memoizeOne from "memoize-one";
+import { fireEvent } from "../../../common/dom/fire_event";
+import { createCloseHeading } from "../../../components/ha-dialog";
+import "../../../components/ha-form/ha-form";
+import { HomeZoneMutableParams } from "../../../data/zone";
+import { haStyleDialog } from "../../../resources/styles";
+import { HomeAssistant } from "../../../types";
+import { HomeZoneDetailDialogParams } from "./show-dialog-home-zone-detail";
+
+const SCHEMA = [
+  {
+    name: "location",
+    required: true,
+    selector: { location: { radius: true, radius_readonly: true } },
+  },
+];
+
+class DialogHomeZoneDetail extends LitElement {
```

Comment:
Would it be good if we just had a generic dialog that could accept a schema and display a ha-form? Feels like that could reduce a lot of duplicate boilerplate.

Reply:
Yeah, that is one my wish list for a while now 👍